workspace_info.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Can you instead restore this to `workspace_status.sh` and work on top of the existing script?
Recent changes were made in https://github.com/kubernetes-sigs/k8s-container-image-promoter/pull/301 and https://github.com/kubernetes-sigs/k8s-container-image-promoter/pull/311 and we should maintain the history.

Reply:
Sure thing! In order to keep workspace_status.sh working with Bazel builds, I added an optional parameter to allow for easier variable injection.

Usage: `workspace_status.sh [inject]`

This removes the need to have two separate workspace scripts.